app/assets/stylesheets/main.scss

```diff
@@ -7,6 +7,7 @@ $dropdown-link-hover-bg:                    #d4edfa;  // sets dropdown link hove
 $link-color:                                #0099d3;  // sets link (also accordion hover, selected table cell) color globally
 $navbar-pf-border-color:                    #063451;  // sets 3px border color at top of screen
 $navbar-pf-bg-color:                        #063451;  // sets backgound color of navigation bar
+//$img-bg-navbar:                             "../navbar.png"; // sets a custom background image in the header
```

Comment:
@Fryguy These are optional settings, we'd like customers to just be able to uncomment to use.